sdk/storage/azure-storage-blob/tests/test_quick_query.py

```diff
@@ -100,8 +98,6 @@ def _teardown(self, bsc):
             except:
                 pass
 
-        return super(StorageQuickQueryTest, self).tearDown()
```

Comment:
Can you see if this `_teardown` method is being automatically called? (via a breakpoint or just seeing if the container is being deleted in Live mode). If it is not, can you change the name to `_tearDown` and then see?

This is mostly curiosity but if this isn't being called at all we can delete it all together. Though I do like a test file that cleans up its own containers! So, it would be nice to know if either of these names get automatically called so we can use them in the future.

Reply:
Will discuss at 1-1; but I have not tested et if this is automatically called, but even if it is, I think we lose this functionality as `StorageRecordedTestCase` does not have a `tearDown`, but it seems that `StorageTestCase` did (although to be fair, I didn't dig through `StorageRecordedTestCase` to see if it had an equivalent, I just tried to blindly call it)

Reply:
Update: `_teardown` does get called 😄 (lowercase d)